src/chigame/users/urls.py

```diff
@@ -21,4 +22,8 @@
     path("cancel_friend_invitation/<int:pk>", view=cancel_friend_invitation, name="cancel-friend-invitation"),
     path("user-list/", views.user_list, name="user_list"),
     path("user_history/<int:pk>", views.user_history, name="user-history"),
+    path("password_reset/", auth_views.PasswordResetView.as_view(), name="password_reset"),
+    path("password_reset/done/", auth_views.PasswordResetDoneView.as_view(), name="password_reset_done"),
+    path("reset/<uidb64>/<token>/", auth_views.PasswordResetConfirmView.as_view(), name="password_reset_confirm"),
+    path("reset/done/", auth_views.PasswordResetCompleteView.as_view(), name="password_reset_complete"),
```

Comment:
I am actually getting the following error when i try to reset my password

Reverse for 'password_reset_confirm' not found. 'password_reset_confirm' is not a valid view function or pattern name.

I followed django documentation here https://docs.djangoproject.com/en/4.2/topics/auth/default/#all-authentication-views 

and added the following 
`    path("accounts/", include("django.contrib.auth.urls")),
`
to src/config/urls.py and it seems to work. The line above automatically adds the necessary paths for password resetting. Just note that I changeed accounts/ to account/ because there is already accounts/ path. I was not sure what else to name it.

Reply:
Just implemented, thank you for finding me the resource, I appreciate that a lot.

Reply:
it works now! but I don't think you need lines 29-32 if you just replace 
`  href="{% url 'users:password_reset' %}"` with 
`href="{% url 'password_reset' %}`
because the latter will use the 
`  path("account/", include("django.contrib.auth.urls")),`
that you added